File: C/Assignments/Assignment 0/q8.c
//8. Write a C program to input the length and width of a rectangle and find its perimeter. 

void main()  {
	float l , b ,per ;
	
	printf("enter the length :  ");
	scanf("%f" , &l) ;
	printf("\n enter the breth : ");
	scanf("%f" , &b) ;
	
	
	per = 2 * (l + b) ;
	
	printf("perimeter is %f" , per);


}

File: C to Java/Assignments/Assignment 0/q8.java
import java.util.Scanner;

public class q8 {

    public static void main(String args[]) {

        Scanner sc = new Scanner(System.in);

        float l , b ,per ;
	
	System.out.printf("enter the length :  ");
	l =  sc.nextFloat();

	System.out.printf("\n enter the breth : ");
	b =  sc.nextFloat();

	
	
	per = 2 * (l + b) ;
	
	System.out.printf("perimeter is %f" , per);
        }
}